sportsref/nfl/players.py

```diff
@@ -177,6 +177,15 @@ def college(self):
         return college
 
     @sportsref.decorators.memoize
+    def collegeid(self):
```

Comment:
I don't think I've been clear. For Brady:
p.college() yields u'michigan'
p.collegeid() yields u'tom-brady-1'
The first one is Tom Brady's college's ID for the cfb website. The one I added yields Tom Brady's ID for the cfb website. My method is useful for looking up college info for an existing NFL player.